api/v1alpha1/server_types.go

```diff
@@ -20,6 +20,13 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// BMC defines data about how to talk to the node via ipmitool
+type BMC struct {
```

Comment:
I could have sworn we had this somewhere else before. Not opposing this change, just trying to remember.

Reply:
yep, we totally did. we have it on the MetalMachine spec in the CAPM provider. but that falls over when you move to using MachineDeployments because you have a MetalMachineTemplate associated with it and it's possible that several nodes would have different BMC login info.

Reply:
:+1: